main.go

```diff
@@ -30,9 +30,9 @@ func serveHTTP() {
 }
 
 func udpAddrFromString(addr string) *net.UDPAddr {
-	host, portStr, err := net.SplitHostPort(*statsdListenAddress)
+	host, portStr, err := net.SplitHostPort(addr)
```

Comment:
Oh, this was changed to `statsdListenAddress` in https://github.com/prometheus/statsd_bridge/pull/6/files. May need updating of your PR here?